tests/ignite/contrib/engines/test_common.py

```diff
@@ -408,7 +409,7 @@ def test_setup_tb_logging(dirname):
 
     tb_logger = _test_setup_logging(
         setup_logging_fn=setup_tb_logging,
-        kwargs_dict={"output_path": os.path.join(dirname, "t1")},
+        kwargs_dict={"output_path": Path(dirname) / "t1"},
```

Comment:
Same here, if `dirname` was Path object we could avoid wraping with Path

Reply:
I'm sorry, but where does this dirname become a Path object?

Reply:
dirname is a pytest fixture defined here: https://github.com/pytorch/ignite/blob/4c2b62bf17aabb998b189b2e39afd59928adf7c1/tests/ignite/conftest.py#L12-L16

By the way, pytest already provides a built-in temp folder fixture but somehow we decided to go with our definition...

https://docs.pytest.org/en/6.2.x/fixture.html#back-to-fixtures

Reply:
Thank you for letting me know.
I will change the relevant part as follows.
line 412: `kwargs_dict={"output_path": Path(dirname) / "t1"},`
line 421: `kwargs_dict={"output_path": Path(dirname) / "t2"},`
line 430: `kwargs_dict={"output_path": Path(dirname) / "t3"},`
line 490: `kwargs_dict={"tracking_uri": Path(dirname) / "p1"},`
line 499: `kwargs_dict={"tracking_uri": Path(dirname) / "p2"},`

Reply:
No, the idea is to update `def dirname():` such that it returns a Path and thus you can do directly : `dirname / "t1"`

As for `tracking_uri`, please read https://github.com/pytorch/ignite/pull/2461#issuecomment-1038952863